README.md

```diff
@@ -14,54 +13,39 @@ If you want to use it directly in a browser use a CDN like [Skypack](https://www
 ```javascript
 import urlJoin from 'url-join';
 
-const id = '42'
-const postUrl = urlJoin('/posts', id, '/metadata/');
-// => "/posts/42/metadata"
+// Create a Full URL
+const fullUrl = urlJoin('http://www.google.com', 'a', '/b/cd', '?foo=123', '&bar=456', '#heading-1');
+fullUrl.toString(); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
 ```
 ## You might not need this library
 
-Web Sandard APIs [URL][1] and [URLSearchParams][1] capable of URL operations except manipulating [URL pathnames][2].
-```
-Use this library to manipulate URL pathnames
-                                      👇
-   https  :// www.example.com /posts/42/metadata ?filter=author&page=1 #heading-1
-|-scheme-|    |---domain----| |----pathname----| |------search-------| |--hash--|
-```
+This library was originally created before the [URL][1] and [URLSearchParams][1] APIs were standardized and widely available in popular runtimes such as browsers and Node.js. Depending on your use-case you might want to consider using these standardized APIs over this library.
```

Comment:
Both the `URL` and `URLSearchParams` links now point to the same place. Let's simplify this and only link to the URL API. Also, let's inline this link.

```suggestion
This library was originally created before the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL_API) was standardized and widely available in popular runtimes such as browsers and Node.js. Depending on your use-case you might want to consider using the standardized API over this library.
```